src/StackExchange.Redis/Maintenance/AzureMaintenanceEvent.cs

```diff
@@ -139,7 +139,7 @@ internal async static Task AddListenerAsync(ConnectionMultiplexer multiplexer, C
                         case AzureNotificationType.NodeMaintenanceEnded:
                         case AzureNotificationType.NodeMaintenanceFailoverComplete:
                         case AzureNotificationType.NodeMaintenanceScaleComplete:
-                            await multiplexer.ReconfigureAsync(first: false, reconfigureAll: true, log: logProxy, blame: null, cause: $"Azure Event: {newMessage.NotificationType}").ForAwait();
+                            await multiplexer.ReconfigureAsync(first: false, reconfigureAll: true, log: null, blame: null, cause: $"Azure Event: {newMessage.NotificationType}").ForAwait();
```

Comment:
Since this is a handler, it's going to run long after that LogProxy (from the initial connection) is dead.

I think what we want to do here is get our circular log idea in and expose _that_. By using the `Action<string>` I have in the current code, that should work underneath with no API change, but open to thoughts here!

Reply:
Consider adding a comment in code to explain why we're not passing through `logProxy`?

Reply:
Closing this out since the public method doesn't even take it, so removes the question - I cheated :)